creme/geolocation/apps.py

```diff
@@ -68,4 +73,17 @@ def register_setting_keys(self, setting_key_registry):
             setting_keys.neighbourhood_distance_key,
             # setting_keys.GOOGLE_API_KEY,
             setting_keys.google_api_key,
+            setting_keys.use_entity_icon_key,
+        )
+
+    def register_geomarker_icons(self):
+        from .registry import GeoMarkerIcon, geomarker_icon_registry
+
+        Contact = get_contact_model()
+        Organisation = get_organisation_model()
+
+        geomarker_icon_registry.register(
+            Contact, GeoMarkerIcon(path='geolocation/images/marker-icon.png')
+        ).register(
+            Organisation, GeoMarkerIcon(path='geolocation/images/marker-icon-red.png')
```

Comment:
I do not see these files in the commit (is it just me?)

Reply:
They're already exist for a long time (the red one was never used before).